src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1194,35 +1214,68 @@ namespace WaterToAirHeatPumpSimple {
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         Real64 rhoair;
-        Real64 MixTemp;
-        Real64 MixHumRat;
-        Real64 MixEnth;
-        Real64 MixWetBulb;
-        Real64 SupTemp;
-        Real64 SupHumRat;
-        Real64 SupEnth;
-        Real64 OutTemp;
-        Real64 ratioTDB;
-        Real64 ratioTWB;
-        Real64 ratioTS;
-        Real64 OutAirFrac;
+        Real64 MixTemp;                   // Mixed air temperature at cooling design conditions
+        Real64 MixTempSys;                // Mixed air temperature at cooling design conditions at system air flow
+        Real64 HeatMixTemp;               // Mixed air temperature at heating design conditions
+        Real64 HeatMixTempSys;            // Mixed air temperature at heating design conditions at system air flow
+        Real64 MixHumRat;                 // Mixed air humidity ratio at cooling design conditions
+        Real64 MixHumRatSys;              // Mixed air humidity ratio at cooling design conditions at system air flow
+        Real64 HeatMixHumRat;             // Mixed air humidity ratio at heating design conditions
+        Real64 HeatMixHumRatSys;          // Mixed air humidity ratio at heating design conditions at system air flow
+        Real64 MixEnth;                   // Mixed air enthalpy at cooling design conditions
+        Real64 MixEnthSys;                // Mixed air enthalpy at cooling design conditions at system air flow
+        Real64 MixWetBulb;                // Mixed air wet-bulb temperature at cooling design conditions
+        Real64 RatedMixWetBulb = 0.0;     // Rated mixed air wetbulb temperature
+        Real64 RatedMixDryBulb = 0.0;     // Rated mixed air drybulb temperature
```

Comment:
Any time you add new variables, they should be declared right where they are first used to stay localized to the appropriate scope.  And when you are already in a module, you are welcome to localize the existing function variables as well.